epipearl/endpoints/admin.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+"""http api calls to epiphan pearl."""
+
+
+class Admin(object):
+    """calls to admin api documented in epiphan user guide.
+
+    epiphan pearl user guide:
+    http://www.epiphan.com/userguides/pdfs/Epiphan_Pearl_userguide.pdf
+
+    in general, methods require arguments below:
+    client: epipearl client
+    channel: epiphan channel call refers to
+    params: dict with params to call
+
+    methods return dict below:
+    {'status_code': int http status code,
+     'response_text': string with body of response}
+    """
+
+    @classmethod
```

Comment:
Don't get the use of class methods here. Could this not be just a standard module with functions for getting/setting params + reboot?